WeCantSpell.Hunspell/SpellCheckResult.cs

```diff
@@ -1,17 +1,17 @@
 ﻿namespace WeCantSpell.Hunspell;
 
-public struct SpellCheckResult
```

Comment:
Maybe some kind of ToString or debugger display info would be helpful to users.

Reply:
I added some bits for this use case and it should make things a bit nicer when debugging.